core/templates/pages/exploration-player-page/services/answer-classification.service.ts

```diff
@@ -185,6 +185,66 @@ export class AnswerClassificationService {
     return answerClassificationResult;
   }
 
+  checkForMisspellings(
+      answer: TextInputAnswer, inputs: TextInputRuleInputs
+  ): boolean {
+    const normalizedAnswer = answer.toLowerCase();
+    const normalizedInput = inputs.x.normalizedStrSet.map(
+      input => input.toLowerCase());
+
+    const hasEditDistanceEqualToTwo = (
```

Comment:
Make `getEditDistance` as a separate utility function. Then call that function here and compare its return value (ie the edit distance) with a constant (=2).

Reply:
A utility function in the same file or another?

Reply:
See if there's a common utility service for string-related methods. If not, create one and add this method there.

Reply:
Done.